abci/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "tendermint-abci"
+version = "0.1.0"
+authors = ["Thane Thomson <thane@informal.systems>"]
+edition = "2018"
+description = """
+    tendermint-abci provides a simple framework with which to build low-level
+    applications on top of Tendermint.
+    """
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[[bin]]
+name = "kvstore-rs"
+path = "src/application/kvstore/main.rs"
+required-features = [ "binary", "kvstore-app" ]
+
+[features]
+client = []
+echo-app = []
+kvstore-app = []
+binary = [ "simple_logger", "structopt" ]
+
+[dependencies]
+bytes = "1.0"
+eyre = "0.6"
+log = "0.4"
```

Comment:
What's the reasoning for using `log` instead of `tracing`, like we do in `tendermint-rpc` or `ibc-rs`?

Reply:
I could switch over to `tracing` - are we going to be sticking with `tracing` then for `tendermint-rpc` and `ibc-rs` for the foreseeable future?

Reply:
Yeah, we intend to stick with `tracing` in `ibc-rs`, as it's strictly more powerful than the other crates we've looked at (even if we do not use it to its full potential at the moment).

Reply:
Refactored! :+1: